version_release.txt

```diff
@@ -0,0 +1 @@
+1.1.0
```

Comment:
So we're now gonna have a version.txt with SNAPSHOT and one which is version_release.txt with version release? Would it not be better to just have one and create a redhat release branch with the release version? This is what we do for all dated (and full) releases

Reply:
yes that might be better to be able to just use the release version. Lets do it in the release issue: #1732

Reply:
Yeah sounds good!